back/app/services/side_fx_user_service.rb

```diff
@@ -3,10 +3,6 @@
 class SideFxUserService
   include SideFxHelper
 
-  def before_create(user, current_user)
-    timestamp_registration(user)
-  end
```

Comment:
I would keep this method and just make it empty, and also keep every place that's calling it.

This is debatable, and if it would just be called from one location I think removing it completely would be fine. But in this case, losing when exactly this side effect should be called from the various places really is core to the business logic (were in the system are we creating a user?) that it could be painful to lose if we need new side effects on before_create of a user.